src/components/screens/bookmarks/addBookmark/addBookmark.js

```diff
@@ -105,11 +105,11 @@ const AddBookmark = ({
   }, []);
 
   useEffect(() => {
-    if (account.data.summary) {
-      const username = account.data.dpos?.delegate?.username ?? '';
+    if (account.data?.summary) {
+      const username = account.data?.dpos?.delegate?.username ?? '';
       setFields(
         [{
-          value: account.data.summary.address,
+          value: account.data?.summary.address,
```

Comment:
Same here. Unnecessary null checks only reduce the reliability of the code.

Reply:
yhea this was because deep scan flagged that `account.data` should also be null checked since `account.data?.dpos` was null checked